test/encodings/encodings/combination_encoding.jl

```diff
@@ -0,0 +1,17 @@
+x = rand(3) # xᵢ ∈ [0, 1] for all xᵢ in x
+e_ord = OrdinalPatternEncoding(length(x))
+e_amp = RelativeMeanEncoding(0, 1, n = 3)
+e_firstdiff = RelativeFirstDifferenceEncoding(0, 1, n = 2)
+e_bin = RectangularBinEncoding(FixedRectangularBinning(0, 1, 2))
+es = [e_ord, e_firstdiff, e_amp, e_bin]
+
+e_combo = CombinationEncoding(es)
+
+symbol = encode(e_combo, x)
+@test symbol isa Int
+d = decode(e_combo, symbol)
+@test d isa AbstractVector
+@test length(d) == length(es)
```

Comment:
These aren't really correctness tests, they are only API test. Why don't we have a correctness test here? Since there are analytic t ests for OrdinalPatternEncoding, and hte behavior of `RelativeMeanEncoding` is straightforward to interpret, please add an analytic test that tests exactly that an encoder integer is exactly 15 or whatever number it should be.